graylog2-web-interface/webpack/vendor-module-ids.json

```diff
@@ -542,6 +542,8 @@
       "./node_modules/value-equal/esm/value-equal.js": 5429,
       "./node_modules/warning/browser.js": 670,
       "dll vendor": 338,
+      "javascript/dynamic|./node_modules/@babel/runtime/helpers/extends.js": 434,
+      "javascript/dynamic|./node_modules/@babel/runtime/helpers/interopRequireDefault.js": 836,
```

Comment:
I might be wrong but are these changes really necessary?

Reply:
Looks like it was changed automatically... I don't know why exactly. I will remove it and check